### DISCUSSION TITLE
Sensors with known (?) protocol not discovered

### DISCUSSION THREAD

**Discussion by KDYR** [merbanan/rtl_433#2227] [Q&A] [answered]
I have a number of Ventus temperature/humidity sensors. Some are W155 and some are W044. Both models should allegedly be using protocol no 16 (AlectoV1 Weather Sensor), but only the W155 is seen when I run "rtl_433 -R 16". 
Is there some kind of tweak that has to be done for the W044 to be discovered/decoded? Someone who has successfully logged data from W044? Any tips?
Eagerly hoping for advice.

**Comment by KDYR** [comment]
Should perhaps have mentioned that I also tried to run rtl_433 without any options, and there is still no sign of any W044. Just a lot of sensors from neighbors ;)

**Comment by zuckschwerdt** [comment]
Make sure you can successfully receive the signal: https://triq.org/rtl_433/ANALYZE.html#verify-a-transmission
If you get a good signal and rtl_433 can't decode that, then post a cu8 sample as zip here.

**Reply by KDYR** [reply]
Thanks a lot for hints on how to attack this!
I ran `rtl_433 -S unknown` and got a file that looks like this:
![image](https://user-images.githubusercontent.com/16138808/199829897-a28c109a-faac-4e98-8e2d-190f481a7826.png)
...but I have no idea if that looks sensible. Did I even do the right thing to grab the transmission?
A zip with the sample is attached.
[g001_433.92M_250k.zip](https://github.com/merbanan/rtl_433/files/9932848/g001_433.92M_250k.zip)

**Reply by KDYR** [reply]
Detached the antenna and ran rtl_433 -w 1.cu
Triggered the transmission manually from the sensor a number of times and got this (rather sure the transmissions are those marked)
![image](https://user-images.githubusercontent.com/16138808/199840510-1abdcc13-ae77-4e1c-9386-d975da3afa6a.png)
Can't figure out what happened the the Y-axix 
[1.zip](https://github.com/merbanan/rtl_433/files/9933218/1.zip)

**Comment by zuckschwerdt** [comment, marked as answer]
re axis: the meta data for that is in the filename (`_433.92M_250k.`)

Even at a quick glance, that looks like strong interference. The 434.0M signal will block reception for rtl_433.
Shift the freq band down so that signal is gone, but your sensor is still there. Maybe `-f 433.85M`

**Reply by KDYR** [reply]
Thanks a lot @zuckschwerdt ,
Your replies inspired by to do further investigations and tests, and I have now established:

- That my W044 sensors are broadcasting on 433.748 MHz
- That my W044 are using the "Generic Temperature" protocol no 38, rather than no 16.
- That when I connect my RTL-SDR to a WinPC and run SDRangel or RTL# there is no signal at 434.010 MHz as my previous samples would otherwise indicate. Not even when I turn on the Rpi ZeroW, that hosted the RTL-SDR when recording the first samples, and put the antenna close to that, there is a signal.
- That if I force the frequency down at bit, my Rpi+dongle can capture and decode the W044 signals. 

At this stage I havent found the source of the 434.010 MHz signal. In one of my planned possible final setups the RTL dongle would be connected to a RPi4, so I should probably test if in that setup there is also some interference. Annoying, because it means I will have to stop my smart home controller which is hosted by the RPi4.
I will also have to investigate if I can widen the bandwidth in order to pick up all usual 433.92 MHz sensor signals as well as those from my W044 that are broadcasting 172 kHz below.
I hope anyone else having issues with W044 may be able to find this post. Again, Thanks to @zuckschwerdt
